proto/viam/service/mlmodel/v1/mlmodel.proto

```diff
@@ -91,3 +98,70 @@ enum LabelType {
   // the position of the tensor value in the axis is the label index
   LABEL_TYPE_TENSOR_AXIS = 2;
 }
+
+message FlatTensorDataInt8 {
+  bytes data = 1;
+}
+
+message FlatTensorDataUInt8 {
+  bytes data = 1;
+}
+
+message FlatTensorDataInt16 {
+  // Pack two 16-bit numbers per entry.
+  // TODO: Describe handling for BE
+  repeated fixed32 data = 1 [packed = true];
```

Comment:
I think we can just remove it, but I don't feel strongly. The reason I say to remove it is it's less things for someone else to read that may/may not know if they need to care about what `packed` means

Reply:
I don't mind either way - almost the opposite of Eric's point, it might be a nice way for someone to see and then learn what `packed` means, if they wanted to